lib/ortho_functions.py

```diff
@@ -757,6 +757,9 @@ def calcStats(args, info):
     elif args.format == 'JP2OpenJPEG':   #### add rgb constraint if openjpeg (3 bands only, also test if 16 bit possible)?
         co = '-co "QUALITY=25" '
 
+    elif args.format == 'JPEG':
```

Comment:
I don't think this block is necessary as blank creation options will be caught in the `else` clause.  Still does not hurt.